src/explorer-components/BountyItem/BountyItem.module.scss

```diff
@@ -12,6 +12,8 @@
 .value {
   margin-left: -1rem;
   text-align: right;
+  display: flex;
+  flex-direction: column;
```

Comment:
i added this so that the secondary value would drop down to be aligned with the date info on the bounty item

Reply:
<img width="459" alt="screen shot 2018-09-09 at 6 13 32 pm" src="https://user-images.githubusercontent.com/5934188/45269426-19582c80-b45c-11e8-85db-bdf4af2330e2.png">

This isn't going to result in those elements being aligned. They exist in separate containers, so they'd need to be moved outside and the component structure would need some altering of the structure if we want those to exist on the same line.

Reply:
ah yeah you're right, that is annoying

Reply:
Let's revert this